frontend/ui/widget/inputtext.lua

```diff
@@ -536,6 +620,41 @@ function InputText:getLineCharPos(line_num)
     return char_pos
 end
 
+-- Get start and end positions of the substring
+-- delimited with the delimiters and containing char_pos.
+-- If char_pos not set, current charpos assumed.
+function InputText:getStringPos(left_delimiter, right_delimiter, char_pos)
+    local char_pos = char_pos and char_pos or self.charpos
```

Comment:
luacheck says `variable 'char_pos' was previously defined as an argument on line 626`
(provided as a parameter to the function, so no need for `local` - it already is.)

Reply:
fixed